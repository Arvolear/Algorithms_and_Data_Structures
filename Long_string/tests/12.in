121187521086740599659114908957
5129214436413810737328660659149