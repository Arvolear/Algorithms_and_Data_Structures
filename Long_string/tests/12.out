5250401957500551336987775568106
5008026915327070137669545750192
621596782671312989414765609038831894783128301992740744097593
42